charts/galoy/templates/deployment.yaml

```diff
@@ -103,10 +103,20 @@ spec:
                 secretKeyRef:
                   name: network
                   key: network
-            - name: REDIS_PORT
-              value: "6379"
-            - name: REDIS_IP
-              value: "galoy-redis-master"
+            - name: REDIS_MASTER_NAME
+              value: "mymaster"
+            - name: REDIS_0_SENTINEL_PORT
+              value: "26379"
+            - name: REDIS_0_DNS
+              value: "galoy-redis-node-0.galoy-redis-headless"
+            - name: REDIS_1_SENTINEL_PORT
+              value: "26379"
+            - name: REDIS_1_DNS
+              value: "galoy-redis-node-1.galoy-redis-headless"
+            - name: REDIS_2_SENTINEL_PORT
+              value: "26379"
```

Comment:
if we use the default value we probably don't need to pass anything for the port

Reply:
Yep, removed.